templates/artist/includes/artist_detail_investors.html

```diff
@@ -1,24 +1,89 @@
+{% load staticfiles %}
+{% load humanize %}
+{% load perdiem %}
+
 <div class="tabs-panel" id="investors">
-    <h2>INVESTORS</h2>
+    {% if user.is_authenticated %}
+    <h3>Invest in {{ campaign.project.reason }}</h3>
     <div class="break">
         <hr>
     </div>
+
+    <!-- Invest -->
+    {% if latest_campaign %}
```

Comment:
I created a new invest include and I think it broke something with the investment module. I'm also not exactly sure the best way to create an include inside of an include. Is there something that I'm missing here?